### PR TITLE
NR Browser configuration

### LINKED ISSUE #787: [NR Browser] Set up NR browser agent for the docs site
## Summary 
Set up NR browser agent to be able to send custom events to NR. 

## AC 
* [ ] Set up NR browser agent 
* [ ] Check that it reports to new relic

### PR DESCRIPTION
**Pull request by zstix**
## Description
Adds New Relic browser configuration to track events in our staging account.

## Related Issues(s)
* Closes #787

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -115,6 +115,16 @@ module.exports = {
...
               errorBeacon: 'staging-bam.nr-data.net',
             },
+            production: {
+              instrumentationType: 'proAndSPA',
+              accountId: '10175106',
+              trustKey: '1',
+              agentID: '29883416',
+              licenseKey: '23448da482',
+              applicationID: '29883416',
+              beacon: 'staging-bam-cell.nr-data.net',
```
Comment (jerelmiller) [inline gatsby-config.js:125]: Did these values change recently? I see our `development` config uses `staging-bam.nr-data.net`. I assume this works correctly?

**Review by jerelmiller — APPROVED** [review]
Looks good! Had one small question about the domain. Thanks for getting this in place!

**Comment by jerelmiller** [comment]
One more small question. Are we instrumenting "staging" (i.e. PR builds) or ignoring those?

File: gatsby-config.js (hunk 1 above)
Comment (zstix) [reply gatsby-config.js:125]: I noticed that as well. I took these values straight from what was provided in the NR Browser snippet, which is what the plugin [suggests](https://github.com/newrelic/gatsby-plugin-newrelic#installation) we do. 🤷‍♂️

Comment (jerelmiller) [reply gatsby-config.js:125]: Oh awesome. As long as that is the case, we should be safe. Thanks!

**Comment by zstix** [comment]
Good question. I don't _think_ we plan on instrumenting the PR builds (at least, that wasn't part of the acceptance criteria).

**Pull request merged**